expirationd.lua

```diff
@@ -949,6 +1084,12 @@ local function show_task_list_obsolete(...)
 end
 
 return {
+    cfg     = setmetatable({}, {
+        __index = cfg,
+        __newindex = function() error("Use expirationd.cfg{} instead", 2) end,
+        __call = expirationd_cfg,
+        __serialize = function() return cfg end,
+    }),
```

Comment:
Will it work as expected across [cartridge's role reloads](https://github.com/tarantool/cartridge/blob/21862c7bedc55aa2c213bd003d0fca85acc1b3f2/cartridge/hotreload.lua)?

Reply:
See also [stash.lua](https://github.com/tarantool/crud/blob/8e0065233cfedf56f21c09a8c7c1e62a0d6e8dbb/crud/common/stash.lua) from crud.

Reply:
It is a bit formal way to look on the problem: sure, we can reject such request till #107, but it does not mean that we have no usages in cartridge applications. I would even say the opposite: we have several internal users at least.

The only thing I'm afraid is something mysterious and weird: for example, if the statistics will disappear at all or if it will be counted twice. If the actual behaviour is like "the `expirationd.cfg.metrics` configuration parameter may lie after reload, but reconfiguration always give a proper result", it should be okay till #107.

I'll leave it up to you to think on risks and decide.

Reply:
It should be ok if an user write stop()/start() cartridge code:
```Lua
local function init()
    expirationd.cfg({metrics = true})
end

local function stop()
    expirationd.cfg({metrics = false})
end
```
It may be the reason why it should be disabled by default.